#!/usr/bin/env python
PACKAGE = "ackermann_drive_publisher"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("target_speed", double_t, 0, "", 0.0, -10.0, 30.0)
gen.add("target_acceleration", double_t, 0, "", 0.0, 0.0, 10.0)
gen.add("target_jerk", double_t, 0, "", 0.0, 0.0, 1.0)

gen.add("target_steering_angle", double_t, 0, "", 0.0, -0.87, 0.87)
gen.add("target_steering_angle_velocity", double_t, 0, "", 0.0, 0.0, 1.0)

exit(gen.generate(PACKAGE, "ackermann_drive_publisher", "ackermann_drive_"))
